### PR TITLE
Add profilePerformance method to Ozzie

### PR DESCRIPTION
**Pull request by jorgecoca**
This is the first step to add performance reports to ozzie. With the method `profilePerformance`, we can get the timeline and the summary generated by `FlutterDrive`. 
Next step is to incorporate those to the HTML report.
Last step will be to give it a score and pass the threshold.

### CODE REVIEW

File: lib/zip_generator.dart [hunk 1]
```diff
@@ -23,6 +26,7 @@ class ZipGenerator {
   /// It generates a ZIP file named `ozzie.zip` with the contents inside the
   /// given [rootFolder]
   Future generateZipWithAllGroups({String rootFolder = 'ozzie'}) async {
+    await _deleteFile(File('$rootFolder/ozzie.zip'));
```
Comment (felangel) [inline lib/zip_generator.dart]: 💯

File: lib/ozzie.dart [hunk 2]
```diff
@@ -70,14 +70,35 @@ class Ozzie {
...
+    Future<dynamic> body(),
+  ) async {
+    final timeline = await driver.traceAction(() async => await body());
+    final summary = TimelineSummary.summarize(timeline);
+    await summary.writeSummaryToFile(
+      reportName,
+      destinationDirectory: 'ozzie/$groupName/profiling',
+      pretty: true,
+    );
+    await summary.writeTimelineToFile(
```
Comment (felangel) [inline lib/ozzie.dart]: why are there two `await summary.writeTimelineToFile` calls?

Comment (jorgecoca) [reply lib/ozzie.dart]: They are not the same call: one is `writeSummaryToFile` (small JSON, just the summary) and the other one is `writeTimelineToFile` (this one is the one you can feed to `chrome::tracing` to see the event timeline)

Comment (felangel) [reply lib/ozzie.dart]: oops I'm blind haha 😝

**Pull request merged**